docs/intro/instances.rst

```diff
@@ -0,0 +1,91 @@
+.. _ref_intro_instances:
+
+=========
+Instances
+=========
+
+Let's get to the good stuff. You can spin up an EdgeDB instance with a single
+command.
+
+.. code-block:: bash
+
+  $ edgedb instance create my_instance
+
+This creates a new instance named ``my_instance`` that runs the latest stable
+version of EdgeDB. (EdgeDB itself will be automatically installed if it isn't
+already.) Alternatively you can specify a specific version with
+``--version``.
+
+.. code-block:: bash
+
+  $ edgedb instance create my_instance --version 2.1
+  $ edgedb instance create my_instance --version nightly
+
+We can execute a query against our new instance with ``edgedb query``. Specify
+which instance to connect to by passing an instance name into the ``-I`` flag.
+
+.. code-block:: bash
+
+  $ edgedb query "select 3.14" -I my_instance
+  3.14
+
+Upgrading
+^^^^^^^^^
```

Comment:
I feel like this is a bit too much for "getting started". Especially given that the preferred way is EdgeDB Projects -- maybe the quickstart should only reference `edgedb project` and nothing else. `edgedb instance` is sort of an advanced command that most users don't have to touch.

Reply:
+1 with @1st1, in hindsight this would be better.

We could touch up on `edgedb instance` though somewhere else where we may want to provide more advanced methods of handling things through CLI - but for a getting started page, this should be kept as simple as possible.